pwiz_tools/Shared/Common/Collections/CollectionUtil.cs

```diff
@@ -202,7 +202,7 @@ public static int CompareColumnValues(object o1, object o2)
             {
                 return 1;
             }
-            return Comparer.Default.Compare(o1.ToString(), o2.ToString());
+            return NaturalComparer.Compare(o1.ToString(), o2.ToString()); // Natural Compare 

```

Comment:
Not as useful a comment as it could be - basically just repeating the name of the function call. We want to let the casual reader easily understand what that function does, so something more like 

// Sort in the same manner as Windows Explorer, where for example [z11ab, z2ab] sorts as ]z2ab, z11ab] (because 11 is more than 2)